scripts/update-status-go.sh

```diff
@@ -29,7 +29,7 @@ VERSION_FILE="${GIT_ROOT}/status-go-version.json"
 SCRIPT_FILE="$(basename "$0")"
 
 STATUS_GO_REPO="${STATUS_GO_REPO:=status-go}"
-STATUS_GO_OWNER="${STATUS_GO_OWNER:=status-im}"
+STATUS_GO_OWNER="${STATUS_GO_OWNER:=qfrank}"
```

Comment:
No, do __NOT__ do this. There's a reason it uses the `${X:=x}` format, because `x` is the default if `X` is not set:
```
STATUS_GO_OWNER=qfrank scripts/update-status-go.sh #13095
```
This is how it should be done.

Reply:
thank you @jakubgs